### ISSUE TITLE
Dispense few drops of water on demand

### ISSUE THREAD

**Issue by GaelleChapuis** [int-brain-lab/iblrig#228]
**Is your feature request related to a problem? Please describe.**
At the beginning of a session (and sometimes during) it would be great to be able to dispense a few water reward manually.

**Describe the solution you'd like**
Several solutions:
1) run a specific task in pyBpod, and press button / keyboard command for each water reward delivered
2) include the above command in the main GUI of pybpod (can dispense water only if no task is running)

**Comment by GaelleChapuis** [comment]
Proposed solution: launch a separate task protocol that would dispense water on 'skip trials'

**Comment by anne-urai** [comment]
This was discussed today in the Physiology WG. 

It would be great if we could run a protocol which on-demand delivers a drop of e.g. 3 uL of water.

**Comment by anne-urai** [comment]
To follow up: if there could just be a version of `flush_water` that opens for the time of e.g. 3 uL, that would pretty much do the job.

**Comment by GaelleChapuis** [comment]
It could be a version of flush water with a timer on the valve. One would have to run flush-water-timed multiple times to give multiple drops.

@nbonacchi if designed like this, could this be implemented quickly and released in a mini-release containing just this change ?

Alternatively @anne-urai do you know how to make such timer on the valve?

**Comment by anne-urai** [comment]
easy way to do this (from https://int-brain-lab.slack.com/archives/C2FK1PTDE/p1595891231452700):

tip: in iblrig_params, duplicate the misc_flush_water folder and rename the .py file to _iblrig_misc_flush_waterdrop. on line 10, change valve_on_time to e.g. 0.1 . Then in pybpod create a new setup under iblrig_misc, and add the flush_waterdrop task - this way, you can easily administer a single drop during the waiting period. in my experience, monitoring the video while you’re inserting probes etc, and giving water when the mouse seems to get frustrated, helps a lot to keep them motivated before starting the task
